cookingWebsite/yourCookbook/viewRecipe.php

```diff
@@ -0,0 +1,112 @@
+<?php
+include('config/init.php');
+verifyUserCooking();
+$recipeId = $_REQUEST['recipeId'];
+$loginId = $_SESSION['loginId'];
+echo yourCookbookEchoHtmlHeader("New Recipe");
+?>
+<script>
+function addTagExistingRecipe(recipeId){
+    var recipeIngredientName = $('#recipeIngredientName').val();
+    var submitItem = $('#submitItem').val();
+    console.log(recipeId, submitItem, recipeIngredientName)
+    $.post('/ajax/saveAddedtag.php', {recipeIngredientName:recipeIngredientName, submitItem:submitItem, recipeId:recipeId}, function(Data){
+          if (Data == "failure"){
+            alert("Error");
+            return false;
+        }
+        // $('.addTag').html(Data)
+    });
+    alert("The tag was sucessfully added!");
+    return false;
+}
+function addRating(loginId, recipeId){
+    var rating = $('#rating').val();
+    var submitItem = $('#submitItem').val();
```

Comment:
With how you're naming this, it's not clear what it's supposed to be. `submitItem` doesn't mean much to me, so I'd suggest leaving a comment or naming it something more intuitive. I saw this pattern somewhere else too although now I'm forgetting where.